grasp_generation/scripts/generate_nerf_data_one_object_one_scale.py

```diff
@@ -31,10 +33,10 @@ def main(args: GenerateNerfDataOneObjectOneScaleArgumentParser):
 
     output_nerf_object_path = (
         args.output_nerfdata_path
-        / f"{args.object_code}_{args.object_scale:.2f}".replace(".", "_")
+        / f"{args.object_code}_{args.object_scale:.4f}".replace(".", "_")
```

Comment:
What's the reason for changing to 4? It's not a big deal, but may need to change this across many other files

Reply:
The main reason is selfish: I'm using this datagen pipeline for my other project and the resolution of the object rescaling is often more than 2 decimal places. If needed, we can add an integer flag for precision and make the default 2 so you don't have to modify your code elsewhere.

Reply:
That's fine, we'll work with the 4 decimal places